### DISCUSSION TITLE
How does the Google Play Games Beta affect OpenGapps or MindtheGapps?

### DISCUSSION THREAD

**Discussion by Danfun64** [LSPosed/MagiskOnWSALocal#255] [Q&A] [answered]
A PC version of Google Play Games is now available [here](https://play.google.com/googleplaygames). I'm curious if this conflicts with Windows Subsystem for Android in any way, and in particular with OpenGapps/MindtheGapps. Considering that Google Play Games is available on Windows 10 and WSA kind of isn't, I'm willing to guess "no", but I'd like a second opinion before I installed things.

**Comment by Howard20181** [comment, marked as answer]
No impact, Google Play Games can't even side-load third-party apps.
